template-processors/base/bin/resourceManager.sh

```diff
@@ -80,6 +80,16 @@ function deleteByOldLabels() {
 function createUpdateResources() {
     local owner="$1"
     local timestamp="$(date +%s)"
+    # Check if directory contains only hidden files like .gitkeep, or .gitignore.
+    # This would mean that user purposefully wanted to track an empty directory in git.
+    # https://git.wiki.kernel.org/index.php/Git_FAQ#Can_I_add_empty_directories.3F
+    if [[ -z $(ls "${MANIFEST_DIR}") ]]; then
+        echo "Manifest directory empty, skipping"
+        return
+    elif [[ -z $(find "$MANIFEST_DIR" -regextype posix-extended -iregex '.*\.(ya?ml|json)') ]]; then
```

Comment:
if adding this, could we unify the `find` call between here and l.47, so that both look the same?

Reply:
Thanks!